### PR TITLE
bumpy fwdpy11 to 0.4.0

### PR DESCRIPTION
**Pull request by molpopgen**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by molpopgen** [comment]
@bioconda/core -- has there been a change in the build setup?  The previous `pip install .` is no longer working:

```
pip._internal.exceptions.InstallationError: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.
```

**Comment by dpryan79** [comment]
For some reason unknown to me, conda doesn't seem to be downloading and extracting the source tarball.

**Comment by molpopgen** [comment]
Bizarre--I can copy/paste and wget the link, and the file does indeed download.

File: recipes/fwdpy11/meta.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package:
   name: fwdpy11
-  version: "0.3.2"
+  version: "0.4.0"
 
-source:
-  url: https://files.pythonhosted.org/packages/fb/b1/2b22483ce6cf354fbc2de54ec09a7aed8c90afad1b56bed8356ab4aa29e4/fwdpy11-0.3.2.tar.gz
-  sha256: a3e436b4bd04933679feb38d6ee85e32f74c110ad09974fa0aeb3d4dfc8b06e4
+  source:
```
Comment (epruesse) [inline recipes/fwdpy11/meta.yaml]: Bug above

Need to see why the linter didn’t catch this.

**Comment by molpopgen** [comment]
@epruesse Thanks--I will merge this now.  Fingers crossed that the linux build makes it this time.   The last version bump didn't ever end up available for download--see #14528.

**Pull request merged**